example/app.js

```diff
@@ -10,6 +10,7 @@ class App extends React.Component {
   constructor() {
     super();
     this.state = {
+      disableSwipe: false,
```

Comment:
Probably not worth adding it to the sample, the documentation for this should be enough.